common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -1,241 +0,0 @@
-import { ParsedUrlQuery } from 'querystring';
-import BaseTabs, { TabType } from './SearchTabs.BaseTabs';
-import { font } from '@weco/common/utils/classnames';
-import styled from 'styled-components';
-import Space from '../styled/Space';
-import { useContext, FunctionComponent, ReactElement, useRef } from 'react';
-import { AppContext } from '../AppContext/AppContext';
-import SearchForm from '@weco/common/views/components/SearchForm/SearchForm';
-import { trackGaEvent } from '@weco/common/utils/ga';
-import NextLink from 'next/link';
-import { removeEmptyProps } from '../../../utils/json';
```

Comment:
Minor: this is the only use of `removeEmptyProps`, so you could delete that too.